Makefile

```diff
@@ -17,7 +17,7 @@ EXTERNAL_TOOLS=\
 	github.com/kardianos/govendor \
 	github.com/client9/misspell/cmd/misspell \
 	github.com/golangci/golangci-lint/cmd/golangci-lint
-GOFMT_FILES?=$$(find . -name '*.go' | grep -v pb.go | grep -v vendor)
+GOFMT_FILES?=$$(find . -name '*.go' | grep -v pb.go | grep -v vendor | grep -v testdata)
```

Comment:
Can you detail why this change is necessary?

Reply:
goimports -w $(find . -name '*.go' | grep -v pb.go | grep -v vendor)
./src/github.com/golangci/golangci-lint/test/testdata/notcompiles/typecheck.go:4:1: expected declaration, found fun
./src/golang.org/x/tools/internal/lsp/testdata/channel/channel.go:23:14: expected operand, found ')'
./src/golang.org/x/tools/internal/lsp/testdata/channel/channel.go:25:3: missing ',' in argument list
./src/golang.org/x/tools/internal/lsp/testdata/importedcomplit/imported_complit.go:25:22: expected selector or type assertion, found '}'
./src/golang.org/x/tools/internal/lsp/testdata/importedcomplit/imported_complit.go:26:3: expected '}', found 'EOF'
gmake[1]: *** [Makefile:208: fmt] Error 2

Reply:
I believe this is due to the way you're invoking `gmake` and setting `GOPATH` to the current working directory, would you agree?

```
env GOPATH=${PWD} PATH=${PWD}/bin:${PATH} gmake ...
```

Reply:
Hmm, even when only calling it as `env PATH=${PWD}/bin:${PATH} gmake bootstrap static-dist dev-ui static-assets` I still see the errors.

Reply:
(If I remove the `PATH`, I get `gmake: go-bindata-assetfs: Command not found`)

Reply:
Any update?

Reply:
I believe you can get around this by setting your GOPATH and PATH variables to a different directory. Could you undo this change so we can get the below `static-assets` fix merged in? If there are still issues the the fmt call a new PR can be opened.

Reply:
I opened https://github.com/hashicorp/vault/pull/7979 to revert this particular part of the change.

Reply:
What should GOPATH and PATH be set to?

Reply:
Nevermind, doesn't seem necessary.